index.html

```
@@ -820,22 +820,111 @@ <h2 class="text-4xl md:text-5xl font-extrabold font-headline tracking-tighter mt
+<<<<<<< HEAD
           <h4 class="font-bold mb-4 flex items-center gap-2"><span class="material-symbols-outlined text-primary text-lg">event</span> Upcoming Deadlines</h4>
-          <div class="space-y-4">
-            <div><p class="text-[10px] font-label uppercase tracking-wider text-zinc-400">MAR 31, 2026</p><p class="text-sm font-bold">Proposal Submission Deadline</p><p class="text-xs text-zinc-500">Ensure your final proposals are submitted by 18:00 UTC.</p></div>
-            <div><p class="text-[10px] font-label uppercase tracking-wider text-zinc-400">APR 21, 2026</p><p class="text-sm font-bold">Review Period Starts</p><p class="text-xs text-zinc-500">Mentors begin ranking of proposals.</p></div>
+<div id="dynamic-deadlines-container" class="space-y-4">
+=======
+          <h4 class="font-bold mb-4 flex items-center gap-2">
+            <span class="material-symbols-outlined text-primary text-lg">event</span> Upcoming Deadlines
+          </h4>
+          
+          <div id="dynamic-deadlines-container" class="space-y-4">
```

HTMLHint (1.9.2)

[error] The id value [ dynamic-deadlines-container ] must be unique.

(id-unique)